emission/tests/netTests/TestVisualize.py

```diff
@@ -56,7 +56,8 @@ def setUp(self):
       section['track_points'] = track_pt_array
       self.SectionsColl.save(section)
 
-  def testCommutePopRoute(self):
+  def DonttestCommutePopRoute(self):
```

Comment:
I'll rewrite

On Sun, Mar 27, 2016 at 8:29 PM, shankari <notifications@github.com> wrote:

> In emission/tests/netTests/TestVisualize.py
> <https://github.com/e-mission/e-mission-server/pull/250#discussion_r57542217>
> :
>
> > @@ -56,7 +56,8 @@ def setUp(self):
> >        section['track_points'] = track_pt_array
> >        self.SectionsColl.save(section)
> >
> > -  def testCommutePopRoute(self):
> > +  def DonttestCommutePopRoute(self):
>
> Can we rewrite the test to match the new code? Or have you written new
> tests for the new code?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/e-mission/e-mission-server/pull/250/files/796e173b906dd5b324da5820f8820a2d83ac6bf5#r57542217>
>